.env-example

```diff
@@ -1,38 +1,47 @@
 API_GATEWAY_PORT=80
 COSMOSDB_PORT=3000
 
+FUNCTIONS_ADMIN_REPO=https://api.github.com/repos/pagopa/io-functions-admin/git/ref/heads/
 FUNCTIONS_ADMIN_CONTEXT=./io-functions-admin
 FUNCTIONS_ADMIN_BRANCH=master
```

Comment:
are these BRANCH variables still used?

Reply:
Yes, they are used to get the commit sha in [here](https://github.com/pagopa/io-mock/pull/18/files#diff-9ea21889e9fbc7d82e7dabb0fb280808R8)